src/stream.js

```diff
@@ -356,23 +356,21 @@ EventDispatcher.prototype.dispatch = function(event) {
  * @type {object}
  * @property {string} event='error'
  *   The event type.
- * @property {?number} txnTS
+ * @property {?number} txn
  *   The event's transaction timestamp.
- * @property {Error} data
+ * @property {Error} event
  *   The underlying error.
  */
 
 /**
  * @typedef {Object} Options
- * @property {string[]} [fields=['ref', 'ts', 'action', 'new']]
+ * @property {string[]} [fields=['action', 'document', 'diff', 'prev']]
  *   The fields event fields to opt-in during stream subscription. Possible
  *   options:
- *   * 'ref': The event's document reference;
- *   * 'ts': The event's valid time;
- *   * 'new': The event's new data;
- *   * 'old': The event's old data;
- *   * 'diff': The event's data difference;
- *   * 'action': The event's action.
+ *   * 'action': The action type
+ *   * 'document': The document's data
+ *   * 'diff': The difference between 'document' and 'prev'
+ *   * 'prev': The event's old data
```

Comment:
```suggestion
 *   * 'prev': The event's previews data
```

Reply:
@erickpintor I'm a bit confused by this annotation -- I thought the `prev` field represented the old document data?

Reply:
It is the old documents data. I thought using the word preview would be more consistent with the field named `prev`. I'm fine with "old" too.

Reply:
@erickpintor Ah, okay. I'll go with `previous` cause that sounds more semantic 👍